systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlConfigurationST.java

```diff
@@ -226,6 +236,65 @@ void testConfigurationFileIsCreated() {
 
         assertThat(cruiseControlConfigurationFileContent, not(nullValue()));
     }
+
+    @Order(6)
+    @Test
+    void testConfigurationPerformanceOptions() throws IOException {
+        Container cruiseControlContainer = null;
+        EnvVar cruiseControlConfiguration = null;
+
+        Map<String, String> kafkaSnapShot = StatefulSetUtils.ssSnapshot(KafkaResources.kafkaStatefulSetName(CLUSTER_NAME));
+        Map<String, String> cruiseControlSnapShot = DeploymentUtils.depSnapshot(CruiseControlResources.deploymentName(CLUSTER_NAME));
+        Map<String, Object> performanceTuningOpts = new HashMap<String, Object>() {{
+                put(CruiseControlServerParameters.CONCURRENT_INTRA_PARTITION_MOVEMENTS.getName(),
+                    CruiseControlServerParameters.CONCURRENT_INTRA_PARTITION_MOVEMENTS.getDefaultValue());
+                put(CruiseControlServerParameters.CONCURRENT_PARTITION_MOVEMENTS.getName(),
+                    CruiseControlServerParameters.CONCURRENT_PARTITION_MOVEMENTS.getDefaultValue());
+                put(CruiseControlServerParameters.CONCURRENT_LEADER_MOVEMENTS.getName(),
+                    CruiseControlServerParameters.CONCURRENT_LEADER_MOVEMENTS.getDefaultValue());
+                put(CruiseControlServerParameters.REPLICATION_THROTTLE.getName(),
+                    CruiseControlServerParameters.REPLICATION_THROTTLE.getDefaultValue());
+            }};
+
+        KafkaResource.replaceKafkaResource(CLUSTER_NAME, kafka -> {
+
+            LOGGER.info("Changing cruise control performance tuning options");
+
+            CruiseControlSpec cruiseControl = new CruiseControlSpecBuilder()
+                    .addToConfig(performanceTuningOpts).build();
+
+            kafka.getSpec().setCruiseControl(cruiseControl);
+        });
```

Comment:
```suggestion
        KafkaResource.replaceKafkaResource(CLUSTER_NAME, kafka -> {
        
            LOGGER.info("Changing cruise control performance tuning options");
            kafka = KafkaBuilder(kafka)
                .editSpec()
                  .editCruiseControl()
                    .addToConfig(performanceTuningOpts)
                  .endCruiseControl()
                .endSpec()
                .build
        });
```

I haven't ran this locally so unsure if it actually works, but this would be slightly more readable and guarantees the prior cruiseControl configuration is not lost (as we are not doing a replace but an edit)

Reply:
Ok, I will try this.

Reply:
I have tried it, and rolling update never happened. No idea why. I am going with:
```
KafkaResource.replaceKafkaResource(CLUSTER_NAME, kafka -> {
            LOGGER.info("Changing cruise control performance tuning options");
            kafka.getSpec().setCruiseControl(new CruiseControlSpecBuilder()
                    .addToConfig(performanceTuningOpts)
                    .build());
        });
```